src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -0,0 +1,136 @@
+@import '../../../common/variables';
+
+$success-color: $shady-acres;
+$error-color: $red;
+
+.be .bcs-task {
+    .bcs-task-assignment-container {
+        margin-left: 40px;
+    }
+
+    .bcs-task-due-date {
+        color: $threes;
+        font-weight: bold;
+        margin: 5px 0;
+
+        span:first-child {
+            padding-right: 3px;
+        }
+
+        &.bcs-task-overdue {
+            color: $error-color;
+        }
+    }
+
+    .bcs-task-assignments {
+        display: flex;
+        margin: 8px 0;
+    }
+
+    .bcs-task-assignment-avatar {
+        height: 32px;
+        width: 32px;
+    }
+
+    .bcs-task-assignment-count {
+        align-items: center;
+        background: $grey;
+        border-radius: 100%;
+        color: #fff;
+        display: flex;
+        justify-content: center;
+    }
+
+    .bcs-task-assignment-status {
+        display: inline-block;
+        height: 32px;
+        position: relative;
+        width: 32px;
+
+        &:not(:last-child) {
```

Comment:
Sibling selectors will allow you to avoid `:not`:

```
& + & {
    margin-left: 3px;
}
```

Reply:
Doesn't that select all but the _first_ element, not the _last_?

Reply:
Yes which is why I switched to `margin-left` in the example above.